src/renderer/ducks/validator/sagas.ts

```diff
@@ -62,13 +50,16 @@ function* loadValidatorsSaga(): Generator<
         const validatorArray: IValidator[] = yield Promise.all(
             validators.map(async (keyStore, index) => {
                 const beaconNodes = await database.validatorBeaconNodes.get(keyStore.getPublicKey());
+                const network = auth.getValidatorNetwork(keyStore.getPublicKey());
+                const balance = await getValidatorBalance(keyStore.getPublicKey(), network, beaconNodes?.nodes[0]);
                 return {
                     name: keyStore.getName() ?? `Validator - ${index}`,
                     status: undefined,
                     publicKey: keyStore.getPublicKey(),
-                    network: auth.getValidatorNetwork(keyStore.getPublicKey()),
+                    network,
+                    balance,
                     keystore: keyStore,
-                    isRunning: undefined,
+                    isRunning: false,
```

Comment:
Aren't you suppose to take this value depending on if lodestar-validator is running or not?

Reply:
Definitely loadstar validator is not running, so seeing no point to keep status undefined if we are sure he is not started and another case `undefined` is a falsy value at the end is same result

Reply:
do we change it somewhere to true?

Reply:
after starting a validator